numpy/core/_add_newdocs.py

```diff
@@ -5615,6 +5714,17 @@
 
     .. versionadded:: 1.13.0
 
+    Examples
+    ---------
+
+    >>> x = np.dtype('8f')
```

Comment:
`np.dtype((float, 8))` I think would be clearer. Let's not require the user to understand the multitude of different string mini-languages to grok these examples..

Can you add another test of `np.dtype((float, (3, 4)))` or similar, which constructs a 2d subarray dtype?